test/pylib/scylla_cluster.py

```diff
@@ -32,12 +34,25 @@
 from cassandra.cluster import EXEC_PROFILE_DEFAULT  # pylint: disable=no-name-in-module
 from cassandra.policies import WhiteListRoundRobinPolicy  # type: ignore
 
-#
-# Put all Scylla options in a template file. Sic: if you make a typo in the
-# configuration file, Scylla will boot fine and ignore the setting.
-# Always check the error log after modifying the template.
-#
-SCYLLA_CONF_TEMPLATE = """cluster_name: {cluster_name}
+
+def make_scylla_conf(workdir: pathlib.Path, host_addr: str, seed_addrs: List[str], cluster_name: str) -> dict[str, object]:
+    core = {
+        'cluster_name': cluster_name,
+        'workdir,W': str(workdir.resolve()),
```

Comment:
I'm surprised this this "workdir,W" name even works... The name of the configuration is supposed to be "workdir" - I think the "W" is its short command-line version.

Reply:
I tried `workdir` and it didn't work